build_scripts/prep_external_site.sh

```diff
@@ -10,8 +10,17 @@ if [[ -z "${EXTERNAL_SITE_PATH}" ]]; then
   echo "EXTERNAL_SITE_PATH not set"
   exit 1
 else
+  THEME_DIR=$(pwd)
   cd $EXTERNAL_SITE_PATH
-  hugo mod tidy
+  if test -f "go.mod"; then
+    mv go.mod go.mod.bak
+  fi
+  touch go.mod
+  printf "module github.com/mitodl/ocw-website\n\n" >> go.mod
```

Comment:
My instinct was to leave it to be more generic because I'm also working on a separate script that I'll make another PR for to replace the `build.sh` script from `ocw-course-hugo-starter` to iterate course builds, but either way it doesn't really matter as long as the site itself isn't inherited as a Hugo module elsewhere.  Either way, we can leave it as `ocw-www` for now.